solutions/cses-1625.mdx

```diff
@@ -0,0 +1,343 @@
+---
+id: cses-1625
+source: CSES (TODO -- convert to something like `USACO Silver 2017 January`)
```

Comment:
```suggestion
source: CSES
```

Reply:
^^ why was this resolved?

Reply:
Oh, I thought I had committed the suggestion... my bad.